packages/p2p-v2/tsconfig.json

```diff
@@ -9,5 +9,5 @@
         }
     },
     "include": ["src", "../../utils.d.ts"],
-    "exclude": ["../shared/**/*"]
+    "exclude": ["../shared/**/*", "./node_modules/*"]
```

Comment:
this tells tsconfig to ignore resolutions for node_modules inside `src`, otherwise it will try to find @deriv/* inside `./src` folder
Like for instance, in this case its trying to find deriv/api inside src folder
<img width="1041" alt="Screenshot 2024-02-06 at 12 30 29 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/15b9aee1-9a77-4f8b-9dee-a9e5a51f4451">